docs: stop excluding rpk ai connection from the rpk reference (#1869)
redpanda-data/documentation: docs/deploy/deployment-option/cloud/managed-connectors/create-gcp-bigquery-connector.mdx

| **An unexpected error occurred while validating credentials for BigQuery: Failed to create credentials from input stream** | The credentials given as a JSON file in the `Credentials JSON` property are incorrect. Copy a valid key from the Google Cloud service account. |
| **JsonConverter with schemas.enable requires "schema" and "payload" fields** | The connector encountered an incorrect message format when reading from a topic. |
| **JsonParseException: Unrecognized token 'test': was expecting JSON** | During reading from a topic the connector encountered a message that is invalid JSON. |

> | **Streaming to metadata partition of column based partitioning table {table_name} is disallowed.** | Check if `bigQueryPartitionDecorator` property is set to `false`. The property can be checked in connector configuration JSON view. |

Reviewer: ```suggestion
| **Streaming to metadata partition of column-based partitioning table {table_name} is disallowed.** | Check to confirm that the `bigQueryPartitionDecorator` property is set to `false`. You can check the property in the connector configuration JSON view. |
```

> | **Caused by: table: GenericData{classInfo=... insertion failed for the following rows:... no such field:** | Redpanda message contains property not existing in a BigQuery table schema. |

Reviewer: ```suggestion
| **Caused by: table: GenericData{classInfo=... insertion failed for the following rows:... no such field:** | The Redpanda message contains a property that does not exist in a BigQuery table schema. |
```

> | **BigQueryConnectException ... insertion failed for the following rows: ...  [row index 0] (location fieldname[0], reason: invalid): This field: fieldname is not a record.** | Redpanda message contains an array of records, but the BigQuery table expects an array of e.g. strings. |

Reviewer: ```suggestion
| **BigQueryConnectException ... insertion failed for the following rows: ...  [row index 0] (location fieldname[0], reason: invalid): This field: fieldname is not a record.** | The Redpanda message contains an array of records, but the BigQuery table expects an array of strings. |
```

> | **BigQueryConnectException: Failed to unionize schemas of records for the table... Could not convert to BigQuery schema with a batch of tombstone records.** | Redpanda message does not contain schema, so the connector cannot create BigQuery table. Create BigQuery table manually. |

Reviewer: ```suggestion
| **BigQueryConnectException: Failed to unionize schemas of records for the table... Could not convert to BigQuery schema with a batch of tombstone records.** | The Redpanda message does not contain a schema, so the connector cannot create a BigQuery table. Create the BigQuery table manually. |
```
